Templates/VisualStudioForMac/MonoGame.IDE.VisualStudioForMac.csproj

```diff
@@ -2,20 +2,16 @@
   <PropertyGroup>
     <TargetFramework>net471</TargetFramework>
     <BaseOutputPath>..\..\Artifacts\VisualStudioForMac</BaseOutputPath>
-    <AssemblySearchPaths>$(AssemblySearchPaths);{GAC}</AssemblySearchPaths>
+    <DefineConstants>IDE;MONOMAC;MAC</DefineConstants>
   </PropertyGroup>

```

Comment:
Don't forget to also add a <Description> block (which I've been adding and updating in #7264)